### ISSUE TITLE
Add cognitive badge

### ISSUE THREAD

**Issue by ajefts** [appirio-tech/topcoder-app#1240]
Add the cognitive badge so it's available on member profiles.

Here are the design assets:

[Cognitive-Badge.zip](https://github.com/appirio-tech/topcoder-app/files/1402912/Cognitive-Badge.zip)

Please use this version:  

![image](https://user-images.githubusercontent.com/3812270/31836262-a84df8b6-b5a2-11e7-8e2f-a61a1d8d659a.png)

**Comment by ajefts** [comment]
@sushilshinde Here is one for you when  you return.  thanks.

**Comment by sushilshinde** [comment]
@ajefts  this is the place, it will be displayed?

![screenshot-www topcoder-dev com-2017-10-25-13-54-43-343](https://user-images.githubusercontent.com/683231/31988268-2aebc568-b98c-11e7-92ca-d7c212f8dd4f.png)

**Comment by ajefts** [comment]
yes, that is where it will be displayed.  You'll see we have some other single badges there as opposed to the "progressive" badges in the image you attached.

**Comment by ajefts** [comment]
For the achievement data...it's not setup yet.  Let me see about doing that today

**Comment by sushilshinde** [comment]
Ok thanks, I will work on the code then.

**Comment by sushilshinde** [comment]
**Completed Items :** 
Code to show badge committed for this change is on feature branch `cognitive-badge-issue#1240`
**Incomplete Items :** 
1. Need sprite image from @Dara-K 
2. Need achievement value for the Cognitive badge in the achievements payload @ajefts to update on that.

**Comment by ajefts** [comment]
2. it is achievement_id 16, 'IBM Cognitive Community'

**Comment by Dara-K** [comment]
What size do you need the badge @sushilshinde ?

**Comment by sushilshinde** [comment]
@Dara-K I sent details on slack.

**Comment by Dara-K** [comment]
Can you send me the details here pls? I don't see them on Slack

**Comment by vic-topcoder** [comment]
Updated assets
![badge-cognitive](https://user-images.githubusercontent.com/8484061/32285521-c73577a6-bee7-11e7-8e87-e5bbaf57bdcc.png)
![badge grid large](https://user-images.githubusercontent.com/8484061/32285522-c747be34-bee7-11e7-98cc-c5cb4180c410.png)

**Comment by sushilshinde** [comment]
@Dara-K got help from @vic-topcoder.
@ajefts need badge data for users TonyJ and _sushilshinde in achievements array.

**Comment by Dara-K** [comment]
Thank you Vic!
